### PR TITLE
[10.0][ADD] web_sheet_widescreen

### PR DESCRIPTION
**Pull request by markrobinson85**
Forked web_sheet_full_width to instead use a media query to increase the width of sheets for widescreens, while maintaining the comfort of using a sheet with negative space around it. Can be easily extended to support more display types/sizes.

### CODE REVIEW

File: web_sheet_widescreen/__manifest__.py [hunk 1]
```diff
@@ -0,0 +1,18 @@
+# -*- coding: utf-8 -*-
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+{
+    "name": "Web - Sheets for Widescreen",
+    "version": "10.0.1.0.1",
+    "author": "Mark Robinson",
```
Comment (nikul-serpentcs) [inline web_sheet_widescreen/__manifest__.py]: @markrobinson85 author like [this](https://github.com/OCA/maintainer-tools/blob/master/template/module/__manifest__.py#L12)

Add 'website' key like [this](https://github.com/OCA/maintainer-tools/blob/master/template/module/__manifest__.py#L11)

File: web_sheet_widescreen/view/qweb.xml [hunk 2]
```diff
@@ -0,0 +1,10 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<openerp>
```
Comment (nikul-serpentcs) [inline web_sheet_widescreen/view/qweb.xml]: @markrobinson85 use `<odoo>` and remove `<data>`

**Review by nikul-serpentcs — COMMENTED** [review]
Improve Code

**Review by hbrunn — CHANGES_REQUESTED** [review]
I think you shouldn't add a new module for this, but change the existing one: Add an extra group, add a config wizard item to enable this group for all employees, and add a template that sets some class on the `o_form_sheet` element. Use this class to select the element then, the net effect is that we can turn this behavior on and off with the config wizard and you barely have to write code for this.
Then I think your CSS is very specific to your setting, aren't we better off with some `calc` expression combined with media queries?

File: web_sheet_widescreen/__manifest__.py (hunk 1 above)
Comment (hbrunn) [reply web_sheet_widescreen/__manifest__.py]: and OCA as author

**Pull request closed without merging**